package.json

```diff
@@ -28,7 +28,7 @@
     "@empathyco/x-adapter": "^8.0.0-alpha.17",
     "@empathyco/x-adapter-platform": "^1.0.0-alpha.58",
```

Comment:
Now that I see the fade better, it seems that it is making all the content of the sliding panel brighter.
It was like that before too, but, should we modify it in another task to only affect the borders?

https://user-images.githubusercontent.com/75546736/214264941-7f2529b8-d49f-44e5-b843-4cf7f050e949.mov

Reply:
IMO I would create a different task to address this.